Utilities/DataLoading/EdFi.BulkLoadClient.Console/readme.md

```diff
@@ -20,5 +20,10 @@
 |      x       |       xsd        | Path to a folder containing the Ed-Fi Data Standard XSD Schema files                                      |
 |      y       |       year       | The target school year for the web API (e.g., 2018)                                                       |
 
+## New for 5.2.0
+* Passing in the base url to the Ods/Api is now required.
```

Comment:
I just realized that I am breaking down stream users, so instead I am removing the required, and will do a check if the flag exists, then it will pull the values from the version information endpoint